app.py

```
@@ -159,10 +159,67 @@ def fetch_mailchimp_lists(api_key: str, server_prefix: str) -> dict[str, Any]:
+        """Clean and format phone number to (###) ### - ####"""
+        try:
+            phone.strip()
+            
+            if pd.isna(phone):
+                return None
+            
+            if len(phone) == 10:
+                return f"({phone[:3]}) {phone[3:6]} - {phone[6:]}"
+            
+            return None
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)